precompile/allowlist/allowlist.go

```diff
@@ -192,6 +192,9 @@ func CreateAllowListPrecompile(precompileAddr common.Address) contract.StatefulP
 
 func CreateAllowListFunctions(precompileAddr common.Address) []*contract.StatefulPrecompileFunction {
 	functions := make([]*contract.StatefulPrecompileFunction, 0, len(AllowListABI.Methods))
+	durangoActivationFunc := func(evm contract.AccessibleState) bool {
+		return evm.GetRules().IsDurangoActivated()
+	}
```

Comment:
Should we move this definition down to be as close as possible to its use within the more narrow scope?